pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -172,6 +175,9 @@ spec:
           - "{{ $cluster }}"
 {{- range $i, $p := $.ContainerPorts }}
 {{- if eq .Protocol "GRPC" }}
+{{- if and $.ProxylessGRPC (ne $p.Port $.GRPCMagicPort) }}
```

Comment:
why not just add an option like LocalhostIP, InstanceIP, etc?

Reply:
Done (in getContainerPorts rather than template)